[DOCS][KSPM] Cloud Posture dashboard - new doc (#2331)
elastic/security-docs: docs/dashboards/cloud-posture.asciidoc

[[cloud-posture-dashboard]]

> = Cloud Posture Dashboard

Reviewer: Making this sentence case, per our guidelines. 
```suggestion
= Cloud Posture dashboard
```

The Cloud Posture dashboard summarizes how your Kubernetes configuration measures up to security benchmarks. 

NOTE: To learn how to collect this data, refer to <<kspm, Kubernetes security posture management>>.

> image::images/cloud-sec-dashboard.png[The Cloud Security dashboard]

Reviewer 2: Add 1 px border to screenshot

> The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the posture of an individual Kubernetes cluster.

Reviewer 3: ```suggestion
The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the posture of an individual Kubernetes cluster.
```

> The first row of cards ("Cloud Posture Score", "Failed Findings", and "Open Cases") summarizes your overall cloud security posture by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the posture of an individual Kubernetes cluster.

Reviewer 3: ```suggestion
The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the posture of an individual Kubernetes cluster.
```

> The Cloud Posture Score card shows the percentage of your findings that passed over time. Hover your mouse to see precisely when the data was collected.

Reviewer 2: I think hovering implies a mouse, so no need to mention it. Also revise to be less vision-oriented:
```suggestion
The Cloud Posture Score card shows the percentage of your findings that passed over time. Hover over the card to display when the data was collected.
```

> The Cloud Posture Score card shows the percentage of your Findings that passed over time. Hover your mouse to see precisely when the data was collected.

Reviewer 3: ```suggestion
The Cloud Posture Score card shows the percentage of your findings that passed over time. Hover your mouse to see precisely when the data was collected.
```

Reviewer 3: In a future doc update, it might be helpful to define what findings are in the context of this dashboard/feature. :)

> The Failed Findings card shows failed findings grouped by CIS Section. Click any of the section names to view the failed findings from that section on the <<kspm, Findings page>>.

Reviewer 2: I don't know if the audience will immediately know what a "CIS Section" is. Wouldn't hurt to at least spell it out, maybe even explain a bit more detail/context:
```suggestion
The Failed Findings card shows failed findings grouped by Center for Internet Security (CIS) benchmark categories. Click any section name to view its failed findings on the <<kspm, Findings page>>.
```
